### PR TITLE
Task/rider on signal

### PR DESCRIPTION
**Pull request by AhmedGhayada23**
install packages :
  firebase_core: ^1.21.1

work 👍 

Add FireBase on the project .
Create OnSignal with a link to Firebase .
Receive Notification with offer .

### CODE REVIEW

**Review by fahjan — APPROVED** [review]

**Review by mohammedalian — COMMENTED** [review]
resolve error

**Review by moaltawil — COMMENTED** [review]
great

File: pubspec.lock [hunk 1]
```diff
@@ -1,6 +1,20 @@
 # Generated by pub
```
Comment (mohammedyousef2) [inline pubspec.lock:1]: add file to .gitignore

**Comment by yonesJr** [comment]
good job

**Comment by azizalshrafi** [comment]
resolve error

**Comment by yonesJr** [comment]
> install packages : firebase_core: ^1.21.1
> 
> work 👍
> 
> Add FireBase on the project . Create OnSignal with a link to Firebase . Receive Notification with offer .

**Pull request merged**